coverage_assessment.R

```diff
@@ -42,6 +42,140 @@ nut <- get_data()
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
 
+SEX <- ifelse(nut$sex == 1, "Male", "Female") ##Assigning male and female to sex
+
+OEDEMA <- ifelse(nut$oedema == 1, "Yes", "No") ## Assigning yes and No to oedema
+
+COUNTY <- nut$county ##Assigning nut$county to COUNTY
+
+Den <- sum(nut$age >=6 & nut$age<60, na.rm = TRUE) ## Assigning the total number of children between 6-60 to Den
+
+
+####NUMBER AND PREVALENCE OF MAM CASES
+
+
+Mod_MUAC <- with (nut, muac>= 11.5 & muac <12.5 & age >=6 & age<60) ## Assigning the moderate MUAC criteria for children between 6-60 for Mod_MUAC
+
+MAM <- table (SEX, OEDEMA, Mod_MUAC, COUNTY, useNA = "ifany") ## Assigning MAM to table 
+
+data.frame(MAM) ##CREATING A DATA FRAME FOR TOTAL MAM CASES
+
+
+MAM_num_F1 <- MAM ["Female", "No", "TRUE","Urban Montserrado"] ## TOTAL MAM FEMALES IN URBAN MONTSERRADO
+
+MAM_num_F2 <- MAM ["Female", "No", "TRUE","Grand Bassa"] ## TOTAL MAM FEMALES IN GRAND BASSA
+
+MAM_num_M1 <- MAM ["Male", "No", "TRUE","Urban Montserrado"] ## TOTAL MAM MALES IN URBAN MONTSERRADO
+
+MAM_num_M2 <- MAM ["Male", "No", "TRUE","Grand Bassa"] ## TOTAL MAM MALES IN GRAND BASSA
+
+MAM_number <- data.frame(Sex = c("Female", "Female", "Male", "Male"),County = c("Urban Montserrado", "Grand Bassa", "Urban Montserrado", "Grand Bassa"), Number_of_MAM = c(MAM_num_F1, MAM_num_F2, MAM_num_M1, MAM_num_M2)) ## CREATING DATA FRAME FOR MAM CASES BY SEX AND COUNTY
+
+MAM_number_table <- xtabs(Number_of_MAM ~ Sex + County, data = MAM_number) ##CREATING A TABLE BY SEX AND COUNTY
+
+MAM_number_table ##VIEW MAM CASES BY SEX AND COUNTY
+
+Prev1_MAM <- MAM ["Female", "No", "TRUE","Urban Montserrado"]/ Den ##PREVALENCE OF FEMALE MAM IN URBAN MONTSERRADO
+
+Prev2_MAM <- MAM ["Female", "No", "TRUE","Grand Bassa"]/ Den #PREVALENCE OF FEMALE MAM IN GRAND BASSA
+
+Prev3_MAM <- MAM ["Male", "No", "TRUE","Urban Montserrado"]/ Den #PREVALENCE OF MALE MAM IN URBAN MONTSERRADO
+
+Prev4_MAM <- MAM ["Male", "No", "TRUE","Grand Bassa"]/ Den #PREVALENCE OF FEMALE MAM IN GRAND BASSA
+
+MAM_prevalences <- data.frame(Sex = c("Female", "Female", "Male", "Male"),County = c("Urban Montserrado", "Grand Bassa", "Urban Montserrado", "Grand Bassa"),Prevalence_of_MAM = c(Prev1_MAM,Prev2_MAM,Prev3_MAM,Prev4_MAM)) ## PREVALENCE OF MAM CASES BY SEX AND COUNTY
+
+MAM_prevalences_table <- xtabs(Prevalence_of_MAM ~ Sex + County, data = MAM_prevalences) ##CREATING TABLE FOR MAM PREVALENCE BY SEX AND COUNTY
```

Comment:
Again, you already have the table results in `MAM_number_table` with the numbers you need to get the prevalence estimate. Why do these calculations this way again? You could have done this instead:

```R
## MAM prevalence females
MAM_number_table[1, ] / Den

## MAM prevalence males
MAM_number_table[2, ] / Den

## MAM prevalence Grand Bassa
MAM_number_table[ , 1] / Den

## MAM prevalence Urban Montserrado
MAM_number_table[ , 2] / Den
```